### DISCUSSION TITLE
HawkEye Error

### DISCUSSION THREAD

**Discussion by ladjosky** [NCAR/lrose-core#110] [Q&A]
Hi ! 
I installed and reinstall lrose-core-20220222 more and more times. Each time, The running of HawkEye failed with the error message below : 
`**

No protocol specified
qt.qpa.xcb: could not connect to display :0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, wayland-egl, wayland, wayland-xcomposite-egl, wayland-xcomposite-glx, xcb.

Aborted (core dumped)

**
`
Every help is welcome.

Thanks.

**Comment by leavesntwigs** [comment]
Hello,
Thank you for letting us know about the errors with HawkEye.  Are you
installing lrose-core on a Mac? If not, what is the computer system that
you are using?
Thanks,
Brenda

**Comment by ladjosky** [comment]
Hi !
Thanks for your qui k reply. M'y computer is à LENNOVO i5 running à Linux
Mint 20.3.
I succède to run it yesterday. It's looked like the path of library files
wasn't found or clear by thé système. I exécuté the LD_LIBRARY_PATH more
times. The issue was due to m'y configuration.

Looking now for the ways to display àn old WSR-74C data in live mode.


Best regards.

Le lun. 27 juin 2022, 17:09, leavesntwigs ***@***.***> a
écrit :

**Comment by leavesntwigs** [comment]
Hello,
I am glad to hear that the LROSE installation is working!
Regarding the old WSR-74C data in live mode, you might try using
RadxPrint to first determine if the LROSE tools recognize the WSR-74C data
files.  If the format is unknown, then please send a few example data files
and we can try to adjust RadxConvert to recognize the format.
Sincerely,
Brenda

**Comment by ladjosky** [comment]
Hello !

Thanks for your supports.

I haven't succeeded to run a captured files by RadxPrint.Here are some
files captured in my drive at the link :
https://drive.google.com/file/d/1jR_HMlPrH-g5GNi7S4i7yaXeTpg3xaJB/view?usp=sharing

Thanks again for guiding me.

Le mar. 19 juil. 2022 à 18:06, leavesntwigs ***@***.***> a
écrit :
